### ISSUE TITLE
pip install dependency on pandoc in v1.0.1

### ISSUE THREAD

**Issue by durera** [watson-developer-cloud/python-sdk#324]
Doing a long overdue update from 0.22.0 to 1.0.1 ... well, trying to:

```
pip install --upgrade watson-developer-cloud

Collecting watson-developer-cloud
  Downloading watson-developer-cloud-1.0.1.tar.gz (137kB)
    100% |################################| 143kB 199kB/s
    Complete output from command python setup.py egg_info:
    See http://johnmacfarlane.net/pandoc/installing.html
    for installation options
    ---------------------------------------------------------------

    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "c:\users\david\appdata\local\temp\pip-build-oio7so\watson-developer-cloud\setup.py", line 72, in <module>
        long_description=read_md('README.md'),
      File "c:\users\david\appdata\local\temp\pip-build-oio7so\watson-developer-cloud\setup.py", line 39, in read_md
        return convert(f, 'rst')
      File "C:\Python27\lib\site-packages\pypandoc\__init__.py", line 69, in convert
        outputfile=outputfile, filters=filters)
      File "C:\Python27\lib\site-packages\pypandoc\__init__.py", line 260, in _convert_input
        _ensure_pandoc_path()
      File "C:\Python27\lib\site-packages\pypandoc\__init__.py", line 544, in _ensure_pandoc_path
        raise OSError("No pandoc was found: either install pandoc and add it\n"
    OSError: No pandoc was found: either install pandoc and add it
    to your PATH or or call pypandoc.download_pandoc(...) or
    install pypandoc wheels with included pandoc.

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in c:\users\david\appdata\local\temp\pip-build-oio7so\watson-developer-cloud\
```

Do we really need to set up and install pandoc to install this module?

**Comment by ehdsouza** [comment]
Could you try :
`pip install I`

**Comment by germanattanasio** [comment]
Can you use the `--verbose` option? Maybe we get more information on what the error is.
```
pip uninstall watson-developer-cloud
pip install --verbose watson-developer-cloud
```

**Comment by durera** [comment]
I tracked it down yesterday evening ... In setup.py there is a try/catch for “from pypandoc import convert” which handles when pypandoc module is not installed, and prevents pip install bailing.  

After digging into it I found that I was running “pip install ... “ on an environment where the pypandoc module was already installed, but Pandoc itself was not setup/configured on the path  .. the wdc  setup.py doesn’t handle that case, it calls the convert method in pypandoc and barfs because pypandoc can’t find pandoc.

The workaround that got me going was/is to pip uninstall pypandoc, pip install wdc, pip install pypandoc

installing and adding pandoc to the path so that pypandoc can find it would also work.

I think it would be good if the whole pypandoc step was a no fail, so that in a case where a system with the module but without pandoc on the path won’t fail to install the module because of the inability to convert a readme, or move the pandoc step into the build process out of setup.py.
